README.md

```diff
@@ -427,6 +428,50 @@ func main() {
     client.Close()
 }
 ```
+
+### SQL
+
+InfluxDB Cloud now supports the ability to query with SQL. With this support
+a user can pass in an SQL query to get a Arrow reader and iterate through the
+records.
+
+> :warning: Parameterized Queries are supported only in InfluxDB Cloud. There is no support in InfluxDB OSS currently.
```

Comment:
The warning should probably say that SQL support is in the cloud only

Reply:
Hmmm not sure if this is a cloud thing, but rather an IOx vs. InfluxDBv2 issue...